### ISSUE TITLE
[BUG] Es muy difícil usar el datepicker actual

### ISSUE THREAD

**Issue by joemmanuel** [omegaup/omegaup#4017]
## Comportamiento Esperado
Poner una fecha de nacimiento no debe ser frustrante :) 

## Comportamiento Actual

> Hola Joe,

>le había comentado a Rodrigo que me tardaré más tiempo en poner mi fecha de nacimiento en >omegaUp que en resolver algunos problemas ;-)

>me cansé de regresarle en el 2014 (y eso porque si no me dice que no tengo permiso de usar >omegaUp) imagina, debo darle toda la vuelta hasta 1972! se necesita otro método de entrada >para esto (además, cada vez que me animo y quiero volver a corregir la fecha, el calendario >comienza de nuevo en 2020 así que me rindo)

>saludos

>Franz

**Comment by tvanessa** [comment]
Si debería ser posible editar la fecha como texto.
Mientras se arregla eso, puedes compartirle este gif para que pueda modificarlo? 
Creo que no es intuitivo, Hugo también había tenido problemas con poner fechas
![datePicker2](https://user-images.githubusercontent.com/13296248/82095129-44743400-96b3-11ea-82fc-080fcd033026.gif)

**Comment by carlosabcs** [comment]
Hay alguna razón por la que usemos exactamente ese datepicker @lhchavez ? O podemos ya ver otras opciones como: https://github.com/charliekassel/vuejs-datepicker

**Comment by lhchavez** [comment]
ese datepicker es el nativo del navegador. es una mejora considerable sobre el que estábamos usando antes (así que imagina lo malo que era D: )

**Comment by pabo99** [comment]
Un par de soluciones que podemos usar son:

https://github.com/nathanreyes/v-calendar
https://github.com/mariomka/vue-datetime

La ventaja de estas es que ya están desarrolladas en vue  y tienen muchas opciones configurables.

**Comment by pabo99** [comment]
Indeed, BootstrapVue offers its own DatePicker component, which could be a viable alternative:

https://bootstrap-vue.org/docs/components/form-datepicker

The selection of the `DatePicker` component should prioritize usability, ensuring that users can easily choose the year with minimal interactions.

**Comment by vaidya45** [comment]
Hi, I'm interested in working on this issue. Could you please assign it to me?

**Comment by pabo99** [comment]
Now this issue is assigned to you!

If you have any questions, don’t hesitate to ask us.
